exporter/lokiexporter/exporter.go

```diff
@@ -35,14 +36,23 @@ type lokiExporter struct {
 	settings component.TelemetrySettings
 	client   *http.Client
 	wg       sync.WaitGroup
+
+	lokiExporterFailedToSendLogRecordsDueToMissingLabels metric.Int64Counter
 }
 
 func newExporter(config *Config, settings component.TelemetrySettings) *lokiExporter {
 	settings.Logger.Info("using the new Loki exporter")
 
+	count, _ := metadata.Meter(settings).Int64Counter(
+		"lokiexporter_send_failed_due_to_missing_labels",
+		metric.WithDescription("Number of log records failed to send because labels were missing"),
+		metric.WithUnit("1"),
+	)
```

Comment:
I see this more and more, should we actually generate an "obsrep" equivalent from metadatagen? I will give it a try maybe during the break.